### DISCUSSION TITLE
Affordable GPU for Carla 0.9.15

### DISCUSSION THREAD

**Discussion by diepnt90** [carla-simulator/carla#8016] [Q&A]
Hi all,

Carla 0.9.15 requires at least RTX 3070 but it is quite expensive as I'm a student.

Is there an alternative with a lower price that still runs Carla 0.9.15 acceptably?

It could be AMD RX580 8Gb?

Thank you.

**Comment by o-antill** [comment]
Have you considered running the Carla server in a no render mode before ? That would definately reduce the usage on your GPU if you were to go for a lower grade one. Furthermore you could just run the simulator in low graphics mode. More of this information is detailed here in the carla docs 'https://carla.readthedocs.io/en/latest/adv_rendering_options/'
